assets/out

```diff
@@ -11,26 +9,22 @@ exec 1>&2 # redirect all output to stderr for logging
 source=$1
 
 if [ -z "$source" ]; then
-  echo "usage: $0 <path/to/volume>"
+  echo "usage: $0 <full/path/to/dir>"
```

Comment:
Doesn't actually need to be a full path, does it? If so, I'd write this as `</full/path/to/dir>`